vnc.html

```diff
@@ -323,9 +323,9 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br />VNC</h1>
     </audio>
 
     <!-- begin scripts -->
-    <script src="core/util.js"></script>
-    <script src="app/webutil.js"></script>
-    <script src="app/ui.js"></script>
+    <script src="node_modules/browser-es-module-loader/dist/babel-browser-build.js"></script>
+    <script src="node_modules/browser-es-module-loader/dist/browser-es-module-loader.js"></script>
+    <script type="module" src="app/ui.js"></script>
```

Comment:
Can't these be put at the top now since only the module loader will be loaded before the page?